src/components/Modal/Modal.styled.tsx

```diff
@@ -41,6 +41,33 @@ export const useModalStyles = (
   } as ReactModal.Styles;
 };
 
+export const useEditModalStyles = (theme: ThemeInterface) => {
+  return {
+    overlay: {
+      position: 'fixed',
+      top: 0,
+      left: 0,
+      right: 0,
+      bottom: 0,
+      backgroundColor: 'rgba(0, 0, 0, 0.7)',
+      zIndex: 20,
+    },
+    content: {
+      height: '100vh',
+      border: '1px solid #ccc',
+      background: theme.color.white,
+      WebkitOverflowScrolling: 'touch',
+      borderRadius: '8px',
+      outline: 'none',
+      padding: '2em',
+      paddingBottom: '1em',
+      display: 'flex',
+      flexDirection: 'column',
+      minHeight: '28rem',
+    },
+  } as ReactModal.Styles;
+};
+
```

Comment:
can you use the `customStyles` instead of passing down the `className="edit"`? I tried it locally from your branch, and you can do these:
1. Get rid of the `edit` className prop in `Modal.tsx` and the `useEditModalStyles`
2. Add this to the `ModalProps` interface and `Modal` props:
```
export interface ModalProps {
  ... other stuff
  customStyles?: ReactModal.Styles | undefined;
}

export const Modal = ({
  ..... the other props here
  customStyles = {},
}: ModalProps) => {
```
3. in `EditClassModal.tsx`, pass down this prop to Modal instead of `className="edit"`
```
<Modal
    .... the other props here
    customStyles={{
        content: {
          position: 'unset',
          transform: '0',
          height: '100vh',
        },
      }}
/>
```

The styling tho will make the modal full screen. But i think that's okay, cause I see it cut off on my end 😅 maybe cause my screen is smaller?
<img width="1512" alt="Screen Shot 2022-07-21 at 8 50 52 PM" src="https://user-images.githubusercontent.com/40800353/180340695-daac8a19-7ff3-402c-8281-021d33e975d2.png">

Reply:
Updated!  I used 90vh & vw, this should not cut off I believe. I tested it in smaller screen.